### ISSUE TITLE
racer returned a non-zero exit code: 101 16: 0x778859cd - BaseThreadInitThunk

### ISSUE THREAD

**Issue by CscLambda** [phildawes/racer#394]
Received this error when using Racer with edubkendo's Atom editor plugin on Windows 7. However, "racer complete std::" in cmd works fine.

**Comment by CscLambda** [comment]
I'm not sure if this is an issue with racer or the plugin, so I've opened an issue there as well.

**Comment by netvl** [comment]
I have lots of these errors when typing doc comments in Atom. The notification area in it is literally overflown by error messages from the plugin.

**Comment by phildawes** [comment]
Hi @netvl 
Ah, it sounds like the plugin is calling into racer on every keypress, even when typing doc comments. I hadn't anticipated that. 
It sounds like racer should handle this because I bet other plugins do this too. I'll have a think about how best to handle this situation. Thanks for the issue report

**Comment by phildawes** [comment]
Hi @netvl 
I tried wrapping the completion function in a thread in order to catch panics. Could you try again and see if this helps please?

**Comment by netvl** [comment]
@phildawes, sure, I'll try it when I get to my home computer.

**Comment by CscLambda** [comment]
This fixed the problem on my setup!

**Comment by netvl** [comment]
Indeed, I haven't seen this flood of messages for some time with the latest maser, looks like it was fixed.
